### ISSUE TITLE
How can I add rect tag in svg to set text background color?

### ISSUE THREAD

**Issue by M4Lien** [jpb12/react-tree-graph#30]
Hi, How are you?
I am using your library by viewing Tree view. (react-tree-graph)
I want to set each text to background.
How can I do?

**Comment by jpb12** [comment]
Sorry, I'm not sure what you mean by 'set each text to background'.

**Comment by M4Lien** [comment]
I am sorry, I'll try to explain it more simply.
![graph](https://user-images.githubusercontent.com/43225865/48680700-59b8f100-ebe9-11e8-92ac-0a3c67dfb09e.png)
As you see, it was duplicated between text and path. so I have to solve this issue.
![graph solve](https://user-images.githubusercontent.com/43225865/48680721-866d0880-ebe9-11e8-9579-755281927084.png)
Are you okay? How can I do?

**Comment by jpb12** [comment]
Based on this [stack overflow answer](https://stackoverflow.com/a/31013492/5111146).

```
<Tree ... textProps={{ filter: 'url(#solid)' }}>
	<defs>
		<filter x="0" y="0" width="1" height="1" id="solid">
			<feFlood floodColor="yellow"/>
			<feComposite in="SourceGraphic"/>
		</filter>
	</defs>
</Tree>
```

Alternatively, you could use textProps to reposition the text node to above/below the line.
